docs/data/material/discover-more/related-projects/related-projects.md

```diff
@@ -56,3 +56,7 @@ Feel free to submit a pull request!
 ### Sparkline
 
 - [mui-plus](https://mui-plus.vercel.app/components/Sparkline): A sparkline is a tiny chart that can be used to indicate the trend of a value.
+
+## Admin frameworks
+
+- [React Admin](https://github.com/marmelab/react-admin): A frontend Framework for building data-driven applications running in the browser on top of REST/GraphQL APIs.
```

Comment:
Waiting on https://mui.zendesk.com/agent/tickets/7593 to be resolved to add refine. mui !== material-ui.